keps/sig-node/4815-dra-partitionable-devices/README.md

```diff
@@ -0,0 +1,2078 @@
+components.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No. The this KEP proposes extensions to an existing type, but not a new type itself.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+Yes and No. With the extensions proposed in this KEP, individual
+`ResourceSlices` have additional fields available to them, thus increasing
+their overall signature. However, the purpose of the new `Mixins` abstraction
+is to actually *reduce* the footprint of these objects, not increase them. As
+such, we expect the size of thes eobjects to actually decrease, not increase,
+but it is ultimately up to how 3rd party vendors decide to use them.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+Yes. The time to allocate a device to a claim (and thus schedule the first pod
+that references that claim) will be affected.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+No.
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No.
+
+### Troubleshooting
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+-->
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+Will be considered for beta.
+
+###### What are other known failure modes?
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+Will be considered for beta.
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
+
+Will be considered for beta.
+
+## Implementation History
+
+- Kubernetes 1.32: KEP accepted as "implementable".
+
+## Drawbacks
+
+It adds complexity to the scheduler.
+
+## Alternatives
```

Comment:
you could link to https://github.com/kubernetes-sigs/wg-device-management/issues/20